go.mod

```diff
@@ -151,6 +150,7 @@ require (
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/sergi/go-diff v1.1.0 // indirect
 	github.com/shopspring/decimal v1.2.0 // indirect
+	github.com/smallcase/go-git/v5 v5.4.3
```

Comment:
what license? how popular? well painted? CII best practices?

vs go-git of course

Reply:
same license
not so popular since it is only a single patch to a bug
yes
yes

I understand if adding a fork is not something that argo wants to take up, I just created this PR so that if in the future the go git bug is still not resolved, you may choose to merge this.

Reply:
what about getting the fix contributed up-stream?

Reply:
Did that, one month ago. No reply.

Reply:
I'm asking for up-votes.

Reply:
Can we fork into argo-labs?